bin/server/cmd/root.go

```diff
@@ -130,6 +143,11 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&flags.demoName, "demo-name", "", "name of the demo to use when running in demo mode")
 	rootCmd.PersistentFlags().BoolVar(&flags.debug, "debug", false, "whether to enable debug level logging")
 	rootCmd.PersistentFlags().StringVar(&flags.pgURL, "pg", "", "optional external PostgreSQL instance connection url(must provide dbname); for example postgresql://user:secret@masterhost:5432/dbname?sslrootcert=cert")
+
+	// Cloud backup related flags.
+	// TODO(dragonly): Add GCS usages when it's supported.
+	rootCmd.PersistentFlags().StringVar(&flags.backupBucket, "backup-bucket", "", "bucket where Bytebase stores backup data, e.g., s3:us-west-2//example-bucket. When provided, Bytebase will store data to the S3 bucket.")
```

Comment:
https://www.replibyte.com/docs/datastores#replibyte-configuration-1

Did some research, but still feel this is too arcane. Also this breaks the search if someone wants to search for all s3:// bucket in the codebase and they will use s3://xxxx

I still favor a separate flag backup-region

@dragonly @d-bytebase thoughts?

Reply:
--backup-region LGTM.